emrun.py

```diff
@@ -798,11 +782,6 @@ def find_gpu_model(model):
         return gpu
     return None
 
-  try:
-    import_win32api_modules()
-  except Exception:
-    return []
```

Comment:
This was don't doing anything useful before this function only depends on `winreg` which will always be available.